api/entity/data/health.go

```diff
@@ -0,0 +1,44 @@
+package data
+
+import "time"
+
+type StatusResponse struct {
+	Health    string    `json:"health"`
+	Timestamp time.Time `json:"timestamp"`
+	// Keepers       KeeperStatus  `json:"keepers"` --- IGNORE ---
+	RootKey       RootKeyStatus `json:"root_key"`
+	BackingStore  BackingStore  `json:"backing_store"`
+	FIPSMode      bool          `json:"fips_mode"`
+	SecretsCount  *int          `json:"secrets_count,omitempty"`
+	UptimeSeconds int64         `json:"uptime_seconds"`
+}
```

Comment:
- Where is `StatusRequest` ? 
- Maybe be more explicit and name it as `NexusStatusRequest` because this will be mostly about status of SPIKE Nexus, or maybe `SystemStatusRequest`
- This entity is misplaced. Please move it to `api/v1/reqres`